commit 31891197ecb87e2742aca4828b47451926228841
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2019-08-05T13:02:46+00:00 (9 weeks ago)
    
    OS-7919 fix broken zfs rsend tests
    Reviewed by: Kody Kantor <kody.kantor@joyent.com>
    Approved by: Kody Kantor <kody.kantor@joyent.com>

diff --git a/usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh b/usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh
index 2026cac20a..e38b5a34a1 100644
--- a/usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh
+++ b/usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh
@@ -130,10 +130,10 @@ function constrain_path
 	# Special case links
 	ln -s /usr/gnu/bin/dd $PATHDIR/gnu_dd
 
-	# SmartOS does not ship some required packages by default.
+	# SmartOS does not ship some required commands by default.
 	# Link to them in the package manager's namespace.
 	pkgsrc_bin=/opt/local/bin
-	pkgsrc_packages="sudo truncate python base64"
+	pkgsrc_packages="sudo truncate python base64 shuf sha256sum"
 	for pkg in $pkgsrc_packages; do
 		if [[ ! -x $PATHDIR/$pkg ]]; then
 			rm $PATHDIR/$pkg &&
